examples/nodejs/cache/.eslintrc.json

```diff
@@ -36,6 +36,10 @@
     "prefer-arrow-callback": "error",
     "no-trailing-spaces": "error",
     "quotes": ["warn", "single", {"avoidEscape": true}],
+    "@typescript-eslint/no-unsafe-call": 0,
+    "@typescript-eslint/no-unsafe-assignment": 0,
+    "@typescript-eslint/no-unsafe-member-access": 0,
+    "node/no-extraneous-import": 0,
```

Comment:
needed these to get build passing w/ these librarys. Wondering if should break these into separate project

Reply:
@cprice404 sorry I left comments after tagging u. left comment here
https://github.com/momentohq/client-sdk-javascript/pull/643/files#r1265604775